### PR TITLE
LPS-168120 Add Autom. Test FDSCustomView#CanUpdateName

### PR DESCRIPTION
**Pull request by eltonccdantas**
**Test Plan**

- Given FDS Sample Portlet
- And Given custom tab
- And Given created custom view
- When edit custom view name
- Then name of custom view is updated in select custom view

**Jira Ticket:** 
https://issues.liferay.com/browse/LPS-168120

### CODE REVIEW

**Comment by eltonccdantas** [comment]
ci:test:frontend:nocompile

**Comment by eltonccdantas** [comment]
cc @john-co

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase [hunk 1]
```diff
@@ -191,6 +191,31 @@ definition {
 		}
 	}
 
+	@description = "LPS-168120. Allow users to rename a custom view"
+	@priority = "4"
+	test CanUpdateName {
+		task ("And Given created custom view") {
+
```
Comment (john-co) [inline modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase:198]: remove line break

File: modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase [hunk 2]
```diff
@@ -191,6 +191,31 @@ definition {
...
+			FDSCustomView.createNewView(key_nameView = "TEST");
+
+			Click(locator1 = "FrontendDataSet#SELECT_OPTION_HEADER");
+
+			Click(
+				key_itemName = "Save View",
+				locator1 = "FrontendDataSet#SELECT_OPTION_HEADER");
```
Comment (john-co) [inline modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase:205]: These lines are not needed. Only FDSCustomView.createNewView(key_nameView = "TEST"); is needed within the Given task

**Comment by john-co** [comment]
please see changes in https://github.com/liferay-frontend/liferay-portal/pull/2859

**Pull request closed without merging**